### ISSUE TITLE
CORS, Technical description: Failed to fetch URL

### ISSUE THREAD

**Issue by thesby** [heartexlabs/label-studio#1391]
```
There was an issue loading URL from $videoSource value

Things to look out for:

URL is valid
URL scheme matches the service scheme, i.e. https and https
The static server has wide-open CORS, more on that here
Technical description: Failed to fetch
```

I am using template of "Video Timeline Segmentation". The video can be loaded, but the audio meets `CORS` problem. I am very confused about this problem. The problem must come from `AudioPlus`.

And if I run label-studio on my local machine, there is no CORS problem. But when I run it on my remote server and use a proxy to visit it, the CORS problem appears.

**Environment (please complete the following information):**
 - OS: Linux
 - Label Studio Version 1.2

**Comment by thesby** [comment]
I think there is no `CORS` problem about my video url, since I can play it in `Chrome`, and I can download it with `wget`.

**Comment by makseq** [comment]
@thesby Have you solved this problem? CORS is a very painful always and LS can't solve this issue automatically, unfortunately.

**Comment by thesby** [comment]
No, I don't know why `CORS` happens in my video urls, because they are public.

**Comment by thesby** [comment]
Is it possible to download all videos or audios to local directory where label-studio can read so that we can skip CORS problem ?

**Comment by makseq** [comment]
@thesby You can use local storage, but you should download all files manually.

**Comment by Malikanhar** [comment]
any updates about the CORS issue? I'm having the same problem too
